### DISCUSSION TITLE
Are conda versions automatically updated?

### DISCUSSION THREAD

**Discussion by juanfcocontreras** [quarto-dev/quarto-cli#5328] [Q&A]
Are conda versions automatically updated?

I can't find version 1.3 on conda repositories.

**Comment by cderv** [comment]
@msarahan how does the update happens for the conda package ?

**Reply by msarahan** [reply]
Usually a bot will issue a PR to the feedstock repository, and the maintainers (me, now also @mfisher87) can merge it once the builds have passed. In this instance, @mfisher87 did the work of putting up a PR: https://github.com/conda-forge/quarto-feedstock/pull/7

As you can see from https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=698807&view=logs&jobId=656edd35-690f-5c53-9ba3-09c10d0bea97&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d, It needs some work. The critical line is https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=698807&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=40

So someone needs to look at this patch (https://github.com/conda-forge/quarto-feedstock/blob/main/recipe/0001-win-conda-path-overrides.patch), figure out why it doesn't apply, and submit a PR to fix or remove that patch.

That person can be anyone. Also, if anyone would like the privilege of merging PRs to the feedstock, you can add yourself as a maintainer with a special comment in an issue or PR: https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-add-user-username

I'm not on the Quarto team, and this is a side project for me. It may be a few days before I can get to it. However, I will try to respond to any question you may have, and also to merge any PRs I see.

**Reply by cderv** [reply]
Thanks for the details answer. I am new to this conda ecosystem but I can learn and be more active to next releases. I'll look at the pointers.

@juanfcocontreras PR to subscribed too is https://github.com/conda-forge/quarto-feedstock/pull/7 then. 

As you may guess, with our conda bundles there will be some delay in release - but we'll keep improving and reduce it

**Reply by mfisher87** [reply]
I'm hoping to have time this week to get my PR passing, reviewed, and merged! Like @msarahan I'm also doing this maintenance work as a free time thing, so I won't be lightning fast.

I'm a long-time conda user, but new to contributing to conda-forge, and am finding the documentation and configured automation very impressive. It's much easier to get started than I thought. I'd be happy to support and co-learn with anyone else who is new to maintaining a feedstock :heart:

**Reply by cderv** [reply]
Awesome @mfisher87  ! 

Really no worry on the lightning fast. Plan is probably to have a release on conda forge for each stable, so it should be not that often. I can be the direct link with the quarto team to help on and take action if needed. I will just needed instructions 😅  

So I am very happy you are offering to help to maintain this - really appreciate the expertise - Eager to learn new stuff in the process. 

i'll follow the opened PR - feel free to ping me any time ! thanks

**Reply by mfisher87** [reply]
1.3 is now available through conda-forge :)

**Comment by Kdreval** [comment]
Thank you, everyone, for this important update!
It seems it hasn't made its way to the release yet? The direct reference doesn't work and the [list of files on conda](https://anaconda.org/conda-forge/r-quarto/files) suggests that 1.2 is still the latest version.

```
conda install -c conda-forge r-quarto=1.3
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - r-quarto=1.3

Current channels:

  - https://conda.anaconda.org/conda-forge/linux-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://conda.anaconda.org/bioconda/linux-64
  - https://conda.anaconda.org/bioconda/noarch
  - https://conda.anaconda.org/r/linux-64
  - https://conda.anaconda.org/r/noarch
  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

**Reply by msarahan** [reply]
R-quarto is a different package. https://github.com/conda-forge/r-quarto-feedstock

**Reply by MattF-NSIDC** [reply]
Here's where you can find the newest release! https://anaconda.org/conda-forge/quarto/files
